pkg/Microsoft.ML.Mkl.Redist/Microsoft.ML.Mkl.Redist.nupkgproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
+
+  <PropertyGroup>
+    <Authors>Intel</Authors>
```

Comment:
Don't need to block the PR on this - but we should make sure CELA is fine with `Intel` listed as the author here.

Reply:
Tensor flow Redist did the same.

Reply:
The value used by Tensorflow Redist was specified by CELA.

Reply:
I will check with CELA and if anything needs to be changed I will change it.

---
In reply to: [222125803](https://github.com/dotnet/machinelearning/pull/977#discussion_r222125803) [](ancestors = 222125803)